src/table.ts

```diff
@@ -1587,28 +1438,32 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
           : entryBatch.map(Mutation.parse),
       };
 
-      const retryOpts = {
-        currentRetryAttempt: numRequestsMade,
-        // Handling retries in this client. Specify the retry options to
-        // make sure nothing is retried in retry-request.
-        noResponseRetries: 0,
-        shouldRetryFn: (_: any) => {
-          return false;
-        },
-      };
-
       options.gaxOptions = populateAttemptHeader(
         numRequestsMade,
         options.gaxOptions
       );
+      if (
+        options.gaxOptions?.retry === undefined &&
+        options.gaxOptions?.retryRequestOptions === undefined
+      ) {
+        // For now gax will not do any retries for table.mutate unless
+        // the user specifically provides retry or retryRequestOptions in the
+        // call.
+        // Moving retries to gax for table.mutate will be done in a
+        // separate scope of work.
+        options.gaxOptions.retry = new RetryOptions(
+          [],
+          DEFAULT_BACKOFF_SETTINGS,
+          () => false
+        );
+      }
```

Comment:
If no retry behaviour is specified by `retry` or `retryRequestOptions` then gax will default to not retrying at all in gax for `table.mutate` because custom retry logic is currently handled in this method. This will change in a later PR where we remove the custom table.mutate retry logic and rely on google-gax for retries.

Reply:
added a question about this in the comment above